backend/src/main/java/saci/application/model/ScoreResponse.java

```diff
@@ -0,0 +1,34 @@
+package saci.application.model;
+
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ScoreResponse {
+    private Optional<LevelResponse> currentLevel;
```

Comment:
```suggestion
    private Optional<Level> currentLevel;
```

Why not to use Level class directly here?